node/cli/Cargo.toml

```diff
@@ -21,7 +21,7 @@ structopt = { version = "0.3.21", optional = true }
 # darwinia-network
 bsc-primitives = { path = "../../primitives/bsc" }
 dp-consensus   = { path = "../../primitives/consensus" }
-drml-service   = { path = "../service" }
+service        = { package = "drml-service", default-features = false, path = "../service" }
```

Comment:
`default-features = false` is necessnary here?

Reply:
Yes. Otherwise, we can not control its features.

Reply:
Weird, why the old version can compile without `default-features = false`?

Reply:
Because all the features set to enable by default.